tests/NPSP.py

```diff
@@ -32,6 +39,15 @@ def __init__(self, debug=False):
         self.payment_list= []
         # Turn off info logging of all http requests 
         logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
+        self._init_locators()
+
+    def _init_locators(self):
+        client = self.cumulusci.tooling
+        response = client._call_salesforce(
```

Comment:
I agree but since Cumulus builds don't run with unreleased CCI code, my process has generally been:
1. Add a keyword to Cumulus and prove it works
2. Add it to CumulusCI and make a release
3. Update Cumulus to use the keyword from CCI and remove the keyword in Cumulus.